src/main/java/NopCommerce/Test/VerifyThatUserIsAbleToSearchProduct.java

```diff
@@ -2,9 +2,11 @@
 
 import org.testng.annotations.Test;
 
-public class VerifyThatUserIsAbleToSearchProduct extends BaseClass{
-   @Test
+public class VerifyThatUserIsAbleToSearchProduct extends BaseClass {
+    @Test
     public void searchProduct() {
         pageFactory.getLoginPage().login();
+        pageFactory.getCatalogPage().catalog();
+        pageFactory.getCatalogPage().catalogVerification("//h1", "Admin area demo");
```

Comment:
As per our last discussion we discuss we do not write xpath in test file and still this is a mistake